app/com/lynxanalytics/biggraph/graph_operations/Aggregate.scala

```diff
@@ -271,6 +271,29 @@ trait CompoundDoubleAggregator[From]
   def outputTypeTag(inputTypeTag: TypeTag[From]) = typeTag[Double]
 }
 
+// ElementwiseAggregator executes a given Double aggregator on Vector[Double] attributes,
+// padding them to equal lengths with the configured zero element if needed.
+class Elementwise[Intermediate](val agg: ItemAggregator[Double, Intermediate, Double])
```

Comment:
Pretty!

Reply:
> Pretty!

I was so surprised this worked out like it did. 😄

Reply:
Scala rocks.